src/types/Privacy.ts

```diff
@@ -83,26 +83,4 @@ export type PrivacyType = {
      * @note The debug geography is used only when the app is in debug mode.
      */
     setConsentFlowDebugUserGeography(userGeography: ConsentFlowUserGeography): void;
-
-    /**********************************************************************************/
-    /* Google UMP Automation */
-    /**********************************************************************************/
-
-    /**
-     * Shows the CMP flow to an existing user.
-     * Note that this resets the user’s existing consent information.
-     *
-     * The function returns when the flow finishes showing. On success, returns null. On failure,
-     * returns one of the {@link CmpError} codes.
-     *
-     * @return {Promise<CmpError|null>}
-     */
-    showCmpForExistingUser(): Promise<CmpError | null>;
-
-    /**
-     * Returns true if a supported CMP SDK is detected.
-     *
-     * @return {boolean}
-     */
-    hasSupportedCmp(): Promise<boolean>;
```

Comment:
@santoshbagadi - Can you take a look here? I want to be consistent with Unity. The change in this PR is moving `Privacy.showCmpForExistingUser()` and `Privacy.hasSupportedCmp()` into the `AppLovinMAX` module such that calls will loo like `AppLovinMAX.showCmpForExistingUser()` and `AppLovinMAX.hasSupportedCmp()`.

How does Unity do it? We should keep API design consistent across plugins.

Reply:
On Unity, the methods are in `MaxCmpService` class. 

- To reshow the flow: `MaxSdk.CmpService.ShowCmpForExistingUser(Action<CmpError>)`
- To check if valid CMP exists: `MaxSdk.CmpService.HasSupportedCmp`

Reply:
Thanks @santoshbagadi - Do you think you can provide guidance on the APIs for the other APIs Hiroshi declared in this `Privacy` module/class?

Reply:
(or @JonathanLiuApp)

Reply:
Unity doesn't really have a `Privacy` class/module. So, this pattern is unique to RN/Flutter. 

I think all the HUC, ARU and DNS APIs make sense here. But we should move everything else (under `TERM FLow` section) to `AppLovinMAX`. Note that Unity doesn't have these methods, since the consent flow is enabled in the Integration Manager on Unity.

Reply:
Thanks, then could you approve this PR?  I'll file another PR for moving the following methods out to `AppLovinMAX`.

```
    /**********************************************************************************/
    /* TERM FLow */
    /**********************************************************************************/

    /**
     * @deprecated Use {@link setTermsAndPrivacyPolicyFlowEnabled()} instead.
     *
     * Enables the MAX Terms Flow.
     *
     * @param enabled true to enable the MAX Terms Flow.
     */
    setConsentFlowEnabled(enabled: boolean): void;

    /**
     * Enables the MAX Terms and Privacy Policy Flow.
     *
     * @param enabled true to enable the MAX Terms and Privacy Policy Flow.
     */
    setTermsAndPrivacyPolicyFlowEnabled(enabled: boolean): void;

    /**
     * The URL of your company’s privacy policy, as a string. This is required in order to enable
     * the Terms Flow.
     *
     * @param urlString The URL string to point your company’s privacy policy.
     */
    setPrivacyPolicyUrl(urlString: string): void;

    /**
     * The URL of your company’s terms of service, as a string. This is optional; you can enable
     * the Terms Flow with or without it.
     *
     * @param urlString The URL string to point your company’s terms of service.
     */
    setTermsOfServiceUrl(urlString: string): void;

    /**
     * Set debug user geography. You may use this to test CMP flow by setting this to {@link ConsentFlowUserGeography.GDPR}.
     *
     * @note The debug geography is used only when the app is in debug mode.
     */
    setConsentFlowDebugUserGeography(userGeography: ConsentFlowUserGeography): void;

```